patches/components-history-core-browser-sync-typed_url_sync_bridge_unittest.cc.patch

```diff
@@ -1,16 +1,16 @@
 diff --git a/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc b/components/history/core/browser/sync/typed_url_sync_bridge_unittest.cc
-index cc1f6024e05d8138bc2b12e9023db51bb6b57504..d80fa4dd7c9be25d415d2af94bfd5727c2f66a42 100644
+index 202d83dbe2d976f61ef064b3853078b94ce87424..6d8aa082f642e7862e8986427fa154a666d6cf89 100644
```

Comment:
this `.patch` file in this PR only updates some git references, but doesn't change anything. It should be safe to remove it from the PR.

Reply:
Yes, this is true if we look on one commit only https://github.com/brave/brave-core/pull/16705/commits/1262573f61b3753ed52bf1531675b885767dedd5 

But if we look on changes from the  first two commits there are other changes in patch file:
https://github.com/brave/brave-core/pull/16705/files/1262573f61b3753ed52bf1531675b885767dedd5

I will squash the commits after I get the approval from the reviewers.

I guess, the git indexes were changed when I rebased PR to cr110, but didn't update the patch files.